### PR TITLE
Release/1.7.2

### PR DESCRIPTION
**Pull request by wow-such-code**

### CODE REVIEW

File: .qube.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+qube_version: '2.7.0-SNAPSHOT # <<QUBE_NO_BUMP>>'
+full_name: Jennifer Bödker
+email: jennifer.boedker@qbic.uni-tuebingen.de
+project_name: core-utils-lib
+project_short_description: core-utils-lib. A qube based .
+version: 1.7.2
```
Comment (jenniferboedker) [inline .qube.yml:6]: This is the same version as on master, we probably need to upgrade the version..

File: src/main/groovy/life/qbic/services/connectors/ConsulConnector.groovy [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
-import io.micronaut.http.uri.UriBuilder
+
 import life.qbic.services.Service
 import life.qbic.services.ServiceConnector
 import life.qbic.services.ServiceType
 
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+
```
Comment (jenniferboedker) [inline src/main/groovy/life/qbic/services/connectors/ConsulConnector.groovy:12]: locally my maven cannot resolve this packages. Do they require java8?

Comment (wow-such-code) [reply src/main/groovy/life/qbic/services/connectors/ConsulConnector.groovy:12]: but this is definitely the part we need to downgrade, then it should work

Comment (jenniferboedker) [reply src/main/groovy/life/qbic/services/connectors/ConsulConnector.groovy:12]: ok, how do you suggest to solve this?

Comment (jenniferboedker) [reply src/main/groovy/life/qbic/services/connectors/ConsulConnector.groovy:12]: Java 11 is provided currently but still I cannot resolve the package

Comment (jenniferboedker) [reply src/main/groovy/life/qbic/services/connectors/ConsulConnector.groovy:12]: ahh nvm.. my intellij project settings said java8.

**Comment by jenniferboedker** [comment]
Since we want to clean up this repo now we decided to close this PR and introduce new ones

**Pull request closed without merging**